data.go

```diff
@@ -6,17 +6,28 @@ import (
 
 // SMTPError specifies the error code and message that needs to be returned to the client
 type SMTPError struct {
-	Code    int
-	Message string
+	Code         int
+	EnhancedCode [3]int
```

Comment:
Could we have a special `EnhancedCode` type, so that it's easier to spot constants?

Reply:
bb4b508dadb2603ed121ddfaf791b082ff94fcfe